src/main/java/com/google/sps/servlets/utils/Statistics.java

```diff
@@ -92,7 +95,8 @@ public Statistics(List<UserComment> userCommentList, int topNComments) {
             BigDecimal.valueOf(userCommentList.get(updatingScoreIdx).getScore());
         if ((scorePoint.compareTo(nextPoint) < 0) || nextPoint.compareTo(UPPER_SCORE) == 0) {
           currentFrequency += 1;
-          // TODO: add topNumComments to the priority queue highMagnitudeList
+          addToFixedQueue(
+              userCommentList.get(updatingScoreIdx), highMagnitudeComments, topNumComments);
         } else {
           break;
         }
```

Comment:
I can't comment on line 111, but we should ensure that `convertQueueToList` returns the PriorityQueue in reverse order to ensure that the comments in the list will be sorted in descending order. 

We should also rename the method to make the intention of the method more clear. Maybe something like `convertQueueToDescendingList`?